src/Okta.Sdk/Internal/OktaClientConfigurationValidator.cs

```diff
@@ -29,7 +29,7 @@ public static void Validate(OktaClientConfiguration configuration)
 
             if (!configuration.OktaDomain.StartsWith("https://", StringComparison.OrdinalIgnoreCase))
             {
-                throw new ArgumentNullException(nameof(configuration.OktaDomain), "Your Okta URL must start with https. You can copy your domain from the Okta Developer Console.");
+                throw new ArgumentException(nameof(configuration.OktaDomain), "Your Okta URL must start with https. You can copy your domain from the Okta Developer Console.");
```

Comment:
This change is needed to get the following IT to pass - `Okta.Sdk.IntegrationTests/OktaClientShould.cs`
It expects the `ArgumentException`.